frontend/src/components/GoalCards/GoalCard.js

```diff
@@ -163,11 +163,13 @@ function GoalCard({
       </div>
 
       <div className={internalLeftMargin}>
-        <ObjectiveButton
-          closeOrOpenObjectives={closeOrOpenObjectives}
-          objectiveCount={objectiveCount}
-          objectivesExpanded={objectivesExpanded}
-          goalNumber={goal.goalNumbers.join('')}
+        <ExpanderButton
+          type="objective"
+          ariaLabel="objectives for goal"
+          closeOrOpen={closeOrOpenObjectives}
+          count={objectiveCount}
+          expanded={objectivesExpanded}
+          number={goal.goalNumbers.join('')}
```

Comment:
can we just merge the number and aria-label properties so we don't have to remember to pass in both? for example ```ariaLabel={`objectives for goal ${goal.goalNumbers.join('')}`}```

Reply:
sure